core/ledger/kvledger/kv_ledger.go

```diff
@@ -38,15 +39,17 @@ var logger = flogging.MustGetLogger("kvledger")
 // KVLedger provides an implementation of `ledger.PeerLedger`.
 // This implementation provides a key-value based data model
 type kvLedger struct {
-	ledgerID   string
-	blockStore blkstorage.BlockStore
-	txtmgmt    txmgr.TxMgr
-	historyDB  historydb.HistoryDB
+	ledgerID       string
```

Comment:
At many places, I see that git diff editor shows me the same line as deleted and then added (like this line-42). Please check for added/removed white-spaces in these areas.